DFC.Digital/DFC.Digital.Web.Sitefinity.DfcSearchModuleTests/SearchIndexEnhancers/JobProfileIndexEnhancerTests.cs

```diff
@@ -44,12 +44,12 @@ public class JobProfileIndexEnhancerTests
             A.CallTo(() => salaryCalculator.GetExperiencedSalary(A<JobProfileSalary>._)).Returns(2000);
 
             var enhancer = new JobProfileIndexEnhancer(fakeJobProfileRepo, fakeJobProfileCategoryRepo, salaryService, salaryCalculator);
-            enhancer.Initialise(dummyJobProfileIndex);
+            enhancer.Initialise(dummyJobProfileIndex, false);
```

Comment:
A permutation of this value as a parameter have been ideal

Reply:
Done